### PR TITLE
blas/testblas: clean up Drotmg test and relax tolerance

### PR DESCRIPTION
**Pull request by vladimir-ch**
For https://github.com/gonum/netlib/pull/92

Please take a look.

### CODE REVIEW

File: blas/testblas/level1double.go [hunk 1]
```diff
@@ -1731,461 +1732,466 @@ func DrotgTest(t *testing.T, d Drotger, skipExtreme bool) {
...
+			d1Want: 0.9999559282289687,
+			d2Want: 0.9898121986058326,
+			x1Want: 0.03662270484346241,
+			pWant: blas.DrotmParams{
+				Flag: blas.Rescaling,
+				H:    [4]float64{1.6110934624105326e-06, -0.000244140625, 0.000244140625, 1.6276041666666668e-06},
+			},
+		},
+
+		// Test cases with values taken from BLAS/TESTING/dblat1.f in the reference.
```
Comment (vladimir-ch) [inline blas/testblas/level1double.go:2060]: It's not visible in the diff: I modified these test cases to use "exact" values from the reference.

Comment (kortschak) [reply blas/testblas/level1double.go:2060]: Why were we previously not using the exact values?

Comment (vladimir-ch) [reply blas/testblas/level1double.go:2060]: The test cases were added in https://github.com/gonum/gonum/pull/420. It looks like you used the runtime output of a printf statement, so they're practically the same values only in a different representation.

Comment (kortschak) [reply blas/testblas/level1double.go:2060]: Yes. That seems likely. I have no memory of that.

I've gone through the netlib/BLAS tests and I'm convinced that that is what I did.

**Review by kortschak — APPROVED** [review]

**Pull request merged**